docs/using/python.rst

```diff
@@ -32,7 +32,7 @@ very creative, let's call it "blog":
 
 ::
 
-    $ tsuru app create blog python
+    $ tsuru app create blog python --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team